### PR TITLE
fix(components): [autocomplete] can not fetch suggestions after clear

### LINKED ISSUE #6801: [Bug Report] [Component] [autocomplete] el-autocomplete组件远程搜索，清空后再次输入内容，远程搜索内容无法显示
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.31`
- Element Plus Version: `2.1.4`
- Browser / OS: ` Chrome/99.0.4844.8 Windows `
- Build Tool: `Vue CLI`

### Related Component

- `el-autocomplete`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtYXV0b2NvbXBsZXRlXG4gICAgdi1tb2RlbD1cInN0YXRlXCJcbiAgICA6ZmV0Y2gtc3VnZ2VzdGlvbnM9XCJxdWVyeVNlYXJjaEFzeW5jXCJcbiAgICBwbGFjZWhvbGRlcj1cIlBsZWFzZSBpbnB1dFwiXG4gICAgQHNlbGVjdD1cImhhbmRsZVNlbGVjdFwiXG4gICAgY2xlYXJhYmxlXG4gIC8+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmLCBvbk1vdW50ZWQgfSBmcm9tICd2dWUnXG5cbmNvbnN0IHN0YXRlID0gcmVmKCcnKVxuXG5pbnRlcmZhY2UgTGlua0l0ZW0ge1xuICB2YWx1ZTogc3RyaW5nXG4gIGxpbms6IHN0cmluZ1xufVxuXG5jb25zdCBsaW5rcyA9IHJlZjxMaW5rSXRlbVtdPihbXSlcblxuY29uc3QgbG9hZEFsbCA9ICgpID0+IHtcbiAgcmV0dXJuIFtcbiAgICB7IHZhbHVlOiAndnVlJywgbGluazogJ2h0dHBzOi8vZ2l0aHViLmNvbS92dWVqcy92dWUnIH0sXG4gICAgeyB2YWx1ZTogJ2VsZW1lbnQnLCBsaW5rOiAnaHR0cHM6Ly9naXRodWIuY29tL0VsZW1lRkUvZWxlbWVudCcgfSxcbiAgICB7IHZhbHVlOiAnY29va2luZycsIGxpbms6ICdodHRwczovL2dpdGh1Yi5jb20vRWxlbWVGRS9jb29raW5nJyB9LFxuICAgIHsgdmFsdWU6ICdtaW50LXVpJywgbGluazogJ2h0dHBzOi8vZ2l0aHViLmNvbS9FbGVtZUZFL21pbnQtdWknIH0sXG4gICAgeyB2YWx1ZTogJ3Z1ZXgnLCBsaW5rOiAnaHR0cHM6Ly9naXRodWIuY29tL3Z1ZWpzL3Z1ZXgnIH0sXG4gICAgeyB2YWx1ZTogJ3Z1ZS1yb3V0ZXInLCBsaW5rOiAnaHR0cHM6Ly9naXRodWIuY29tL3Z1ZWpzL3Z1ZS1yb3V0ZXInIH0sXG4gICAgeyB2YWx1ZTogJ2JhYmVsJywgbGluazogJ2h0dHBzOi8vZ2l0aHViLmNvbS9iYWJlbC9iYWJlbCcgfSxcbiAgXVxufVxuXG5sZXQgdGltZW91dDogTm9kZUpTLlRpbWVvdXRcbmNvbnN0IHF1ZXJ5U2VhcmNoQXN5bmMgPSAocXVlcnlTdHJpbmc6IHN0cmluZywgY2I6IChhcmc6IGFueSkgPT4gdm9pZCkgPT4ge1xuICBjb25zdCByZXN1bHRzID0gcXVlcnlTdHJpbmdcbiAgICA/IGxpbmtzLnZhbHVlLmZpbHRlcihjcmVhdGVGaWx0ZXIocXVlcnlTdHJpbmcpKVxuICAgIDogbGlua3MudmFsdWVcblxuICBjbGVhclRpbWVvdXQodGltZW91dClcbiAgdGltZW91dCA9IHNldFRpbWVvdXQoKCkgPT4ge1xuICAgIGNiKHJlc3VsdHMpXG4gIH0sIDMwMDAgKiBNYXRoLnJhbmRvbSgpKVxufVxuY29uc3QgY3JlYXRlRmlsdGVyID0gKHF1ZXJ5U3RyaW5nOiBzdHJpbmcpID0+IHtcbiAgcmV0dXJuIChyZXN0YXVyYW50OiBMaW5rSXRlbSkgPT4ge1xuICAgIHJldHVybiAoXG4gICAgICByZXN0YXVyYW50LnZhbHVlLnRvTG93ZXJDYXNlKCkuaW5kZXhPZihxdWVyeVN0cmluZy50b0xvd2VyQ2FzZSgpKSA9PT0gMFxuICAgIClcbiAgfVxufVxuXG5jb25zdCBoYW5kbGVTZWxlY3QgPSAoaXRlbTogTGlua0l0ZW0pID0+IHtcbiAgY29uc29sZS5sb2coaXRlbSlcbn1cblxub25Nb3VudGVkKCgpID0+IHtcbiAgbGlua3MudmFsdWUgPSBsb2FkQWxsKClcbn0pXG48L3NjcmlwdD5cbiIsImltcG9ydC1tYXAuanNvbiI6IntcImltcG9ydHNcIjp7fX0ifQ==)

### Steps to reproduce

官网https://element-plus.gitee.io/zh-CN/component/input.html#%E8%BF%9C%E7%A8%8B%E6%90%9C%E7%B4%A2
1. 增加clearable
2. 输入v,出现下拉菜单,不要选择；
2. 点击清空按钮；
3. 再次输入v,下拉菜单未显示；

### What is Expected?

正常显示

### What is actually happening?

隐藏了

### Additional comments

_(empty)_

动画

### PR DESCRIPTION
**Pull request by buqiyuan**
fix #6801
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

before: 
image

after:
image

### CODE REVIEW

File: packages/components/autocomplete/src/autocomplete.vue [hunk 1]
```diff
@@ -194,6 +194,7 @@ const handleInput = (value: string) => {
   emit('input', value)
   emit(UPDATE_MODEL_EVENT, value)
   suggestionDisabled.value = false
+  activated.value = Boolean(value)
```
Comment (btea) [inline packages/components/autocomplete/src/autocomplete.vue:197]: Maybe no need to add code，just remove this line
https://github.com/element-plus/element-plus/blob/c098496f0a08c3b371eb3dc78ba9265e682e8afb/packages/components/autocomplete/src/autocomplete.vue#L218

Comment (buqiyuan) [reply packages/components/autocomplete/src/autocomplete.vue:197]: yeah, I know this line, but the suggestions should be closed after clear, this behavior is also the same in `element-ui2.x`

Comment (btea) [reply packages/components/autocomplete/src/autocomplete.vue:197]: Okay, I see.

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
LGTM. @btea Do you got any other comments on this?

**Comment by btea** [comment]
Do we need to add tests?

**Comment by JeremyWuuuuu** [comment]
> Do we need to add tests?

ideally we should, but this seems ok, we can do that in an additional PR if @buqiyuan agrees

**Review by btea — APPROVED** [review]

**Comment by btea** [comment]
@JeremyWuuuuu ok.

**Comment by buqiyuan** [comment]
i will do that in additional PR



---Original---
From: ***@***.***&gt;
Date: Fri, Mar 25, 2022 15:25 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [element-plus/element-plus] fix(components): [autocomplete] cannot fetch suggestions after clear (PR #6820)




  
Do we need to add tests?
  
ideally we should, but this seems ok, we can do that in an additional PR if @buqiyuan agrees

**Pull request merged**